rvzr/model_dynamorio/model.py

```diff
@@ -28,9 +28,16 @@
 _DRRUN_CMD: Final[str] = "~/.local/dynamorio/drrun " \
     "-c ~/.local/dynamorio/libdr_model.so {flags} -- {binary} {args}"
 
-_TRACE_TYPE = Literal["eot", "pc", "mem", "reg"]
-_TRACE_ID_TO_NAME: Final[Dict[int, _TRACE_TYPE]] = {0: "eot", 1: "pc", 2: "mem", 3: "mem", 4: "reg"}
+_TRACE_TYPE = Literal["eot", "pc", "mem"]
```

Comment:
There was a code style mistake here. Python types are typically in CamelCase, similar to classes. So this one (and the following types) should be _TraceType